user/urls.py

```diff
@@ -1,9 +1,15 @@
 from django.urls import path
+from rest_framework.routers import SimpleRouter
 
-from user.views import login_view, logout_view, auth_view
+from user.views import *
+
+router = SimpleRouter()
+router.register('users', UserViewSet, 'users')
 
 urlpatterns = [
     path('login/', login_view),
     path('logout/', logout_view),
-    path('auth/', auth_view)
+    path('user/', retrieve_user_view),
```

Comment:
Think having `user/` and `users/` as two subtly different URLs is a bad idea, maybe call the current user one `me/`.

Reply:
e2e58a29